scst-store/security.md

```diff
@@ -0,0 +1,78 @@
+# Security for Supply Chain Security Tools - Store
+
+## Application Security:
+
+### TLS Encryption
+
+The Store requires TLS connection. If certificates are not provided the store will not start. The Store supports TLS v1.2 and TLS v1.3. The Store does not support TLS 1.0, so a downgrade attack cannot happen. TLS 1.0 prohibited under Payment Card Industry Data Security Standard (PCI DSS).
+
+##### Cryptographic Algorthms:
+
+Elliptic Curve:
+```
+CurveP521
+CurveP384
+CurveP256
+```
+
+Cipher Suites:
+```
+TLS_AES_128_GCM_SHA256
+TLS_AES_256_GCM_SHA384
+TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+```
+
+### Access Controls
+
+The Store uses [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy) as the only entry point to the Store's API. Authentication and Authorization must be completed successfully via the `kube-rbac-proxy` before the Store's API accessible.
+
+##### Authorization
+
+The `kube-rbac-proxy` uses [Token Review](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) to verify if the token is valid. `Token Review` is a Kubernetes API to ensure the access token provided by the user was issued by a trusted vendor. To issue an access token using Kubernetes, the user can create a Kubernetes Service Account and retrieve the corresponding generated Secret for the access token.
+
+To create an access token, please refer to the [Create Service Account Access Token Docs](create_service_account_access_token.md)
+
+##### Authentication
+
+The `kube-rbac-proxy` uses [Subject Access Review](https://kubernetes.io/docs/reference/access-authn-authz/authorization/) to ensure the user has access to certain operations. `Subject Access Review` is a Kubernetes API that uses [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to deternmine if the user can perform the current action.
+
+Please refer to the [Create Service Account Access Token Docs](create_service_account_access_token.md)
+
+There are only 2 roles that is supported: Read Only User and Read and Write User.
+
+*** Note: We do not have roles that limited access to resources (Ex: images, packages, vulnerabilities, etc...).
+
+## Container Security
+
+### Non-root User
+All containers shipped do not use root. Using Kubernetes Security Context to ensure that the start up use is not root.
+
+Security Context for the Store:
+    allowPrivilegeEscalation: false
+    runAsUser: 65532
+    fsGroup: 65532
+
+Security Context for the Postgres DB POD:
+    allowPrivilegeEscalation: false
+    runAsUser: 999
+    fsGroup: 999
+
+*** Note: 65532 is the uuid for the "nobody" user. 999 is the uuid for the "postgres" user.
+
+## Security Scanning
+
+There are 2 types of security scans that is performed before every release.
```

Comment:
I believe this should be:
"security scans that is performed" => "security scans that are performed"